src/main/java/org/jabref/gui/maintable/MainTableColumnFactory.java

```diff
@@ -129,27 +129,26 @@ public MainTableColumnFactory(BibDatabaseContext database, ColumnPreferences pre
     }
 
     private Node createGroupColorRegion(BibEntryTableViewModel entry, List<AbstractGroup> matchedGroups) {
-        Color groupColor = matchedGroups.stream()
-                .flatMap(group -> OptionalUtil.toStream(group.getColor()))
-                .findFirst()
-                .orElse(Color.TRANSPARENT);
-
-        if (groupColor != Color.TRANSPARENT) {
-            Rectangle border = new Rectangle();
-            border.setWidth(5);
-            border.setHeight(20);
-            border.setFill(Color.DARKGRAY);
-
-            Rectangle groupRectangle = new Rectangle();
-            groupRectangle.setWidth(3);
-            groupRectangle.setHeight(18);
-            groupRectangle.setFill(groupColor);
-
-            StackPane container = new StackPane();
+        List<Color> groupColors = matchedGroups.stream()
+                                               .flatMap(group -> OptionalUtil.toStream(group.getColor()))
+                                               .collect(Collectors.toList());
+
+        if (!groupColors.isEmpty()) {
+            HBox container = new HBox();
+            container.setSpacing(2);
             container.setMinWidth(10);
             container.setAlignment(Pos.CENTER);
```

Comment:
Is it possible to set it to left align? Looks better imho 😀

Reply:
Good idea, but now I cannot decide what I prefer 😸 

Left-aligned
![image](https://user-images.githubusercontent.com/5037600/65793994-77d77200-e167-11e9-93e8-81307720dd3a.png)

Right-aligned
![image](https://user-images.githubusercontent.com/5037600/65794021-89207e80-e167-11e9-96e2-40783b5252dd.png)

or the centered version from above...?!

Reply:
I would still say left aligned... But that's just my personal opinion 😁